guides/AUCTIONS_WINNERS.md

```diff
@@ -0,0 +1,147 @@
+    "10"
+  ],
+  "winning": [
+    {
+      "bid": null,
+      "leaseSet": [
+        "3"
+      ]
+    },
+    {
+      "bid": {
+        "accountId": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
+        "paraId": "1000",
+        "amount": "500000000000000"
+      },
+      "leaseSet": [
+        "3",
+        "4"
+      ]
+    },
+    {
+      "bid": null,
+      "leaseSet": [
+        "3",
+        "4",
+        "5"
+      ]
+    }
+  ],
+  ...
+}
+```
+
+We will track and store `finishEnd`,`auctionIndex`, and `leasePeriods` in a Database. 
+
+** Details of each key **
+
+`finishEnd`: This is the last block of the auction. Storing it allows you to query the block at which the auction ended. From that block you can extract the lease winning related events. (To query the block: GET `/blocks/{finishEnd}`.)
+
+`auctionIndex`: The unique identifier for the auction. 
+
+`leasePeriods`: The available lease period indexes that may be bid on for the specific `auctionIndex`. 
+
+
+## Walkthrough
+
+### Relay-chain Dev Setup for auctions (SKIP if you already have a development setup)!
+-------------
+
+The begining of this guide will start by briefly introducing setting up a simple parachain enviornment if you dont already know how or dont have one set for local developement. 
+
+Start by cloning [polkadot](https://github.com/paritytech/polkadot) and checking out the 'release-v0.9.5' branch. NOTE: Before compiling make sure to adjust the [EndingPeriod](https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L1158) to `100`, and [LeasePeriod](https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L1123) to `100` so the auction time is fit for local development. You can then follow this [gist](https://gist.github.com/emostov/a58f887fce6af8a9b4aa2421114836c5) to get your alice and bob dev validator nodes up (make sure to change '--chain' to 'kusama'). Then you can call those extrinsic calls with the polkadot-js UI. (Note: When the latest polkadot release contains the upgrade to parachains, the above links will reflect that change)
+
+### Using Sidecar to find the auction winners
+
+`/experimental/paras/auctions/current` ENDPOINT
+
+An ongoing auction is either in one of two phases: `starting` or `ending`. During an ongoing auction the endpoint returns a `finishEnd` key which denotes the block where the `AuctionClosed` and `Leased` events are emitted. The aforementioned events are emitted under the `on_initialize` key. 
+
+By storing the `finishEnd` block and looking at the `Leased` events within it, we can see who the auction winners are and what lease periods they were rewarded.
+
+The next important part is the `leasePeriods` that corresponds to the current `auctionIndex`. We use these available `leasePeriods` to compare the winning results and see which `paraId`'s took which slots. (Note that this is a redundant way to find the auction winners when coupled with watching for the `Leased` events)
```

Comment:
Redundant and less granular - it does not explicitly say which auction they won the slots from and multiple auctions may cover the same set of slots.